lib/sisimai/address.rb

```diff
@@ -138,29 +138,29 @@ def self.find(argv1 = nil, addrs = false)
               # <"neko(nyaan)"@example.org> or <neko(nyaan)@example.org>
               if v[:address] =~ /["]/
                 # Quoted local part: <"neko(nyaan)"@example.org>
-                v[:address] += e
+                v[:address] << e
 
               else
                 # Comment: <neko(nyaan)@example.org>
                 readcursor |= indicators[:'comment-block']
-                v[:comment] += ' ' if v[:comment] =~ /[)]\z/
-                v[:comment] += e
+                v[:comment] << ' ' if v[:comment].end_with?(')')
```

Comment:
__[RuboCop]__

Metrics/BlockNesting: Avoid more than 4 levels of block nesting.


<!-- Comment by [SideCI](https://sideci.com) -->